cell/model/Workbook.js

```diff
@@ -1746,7 +1746,8 @@
 	};
 	Workbook.prototype._generateFontMap=function(){
 		var oFontMap = {
-			"Arial"		: 1
+			"Arial"		: 1,
+			"SimSun"    : 1

```

Comment:
It's unacceptable to make SimSun as default font. This will slow down the opening of each spreadsheet file.

Reply:
The critical problem is that if the font set in *.xlsx for east Asia font doesn't really support the unicode ranges of east Asia chars. with just loading Arial as default font, it won't be possible to display east Asia char properly. We have tested an example xlsx file which set Chinese chars' font to Arial, without loading SimSun font, the initialization of display cannot be complete properly.
Or we can load the corresponding font when we really use it. But we were wondering if it will needed as editor at least loads not only default font but also all the fonts declared in spreadsheet file. Adding just one (one for Chinese, maybe four for all east Asia languages) default font will slow down the opening in an unacceptable level?